documentation/en/source/pages/templates-spec/stack-bundles.rst

```diff
@@ -35,7 +35,7 @@ The valid keys to use within a bundle are:
 * ``category`` (optional): a string providing the category name for the bundle.
 * ``maintainer`` (optional): a string providing the maintainer for the bundle (if not provided, the user loginName will be used).
 * ``website`` (optional): a string providing the website URL for the bundle or maintainer.
-* ``oses`` (optional): a list of os for which the bundle is designed (please refer :ref:`stack-os` for more information, only name, version and arch is necessary).
+* ``restrictionRule`` (optional): a string providing the restriction rule of the bundle (For which distributions or target formats (e.g. builder's types, see :ref:`template-builders` section) the bundle is designed for or not).
```

Comment:
I would suggeste: a string which indicates the "restriction rule" for a bundle. A restriction rule lists which distributions and/or target formats the bundle is designed for.